src/main/java/domain/store/WinningMoney.java

```diff
@@ -0,0 +1,47 @@
+package domain.store;
+
+import java.util.Objects;
+
+public class WinningMoney {
+	private static final int PERCENT = 100;
+
+	private long winning;
+
+	public WinningMoney(long winning) {
+		validate(winning);
+		this.winning = winning;
+	}
+
+	private void validate(long winning) {
+		if (winning < 0) {
+			throw new IllegalArgumentException("총 당첨 금액은 음수일 수 없습니다.");
+		}
+	}
+
+	public static WinningMoney sum(WinningMoney winningMoney, WinningMoney winningMoney1) {
```

Comment:
```suggestion
	public static WinningMoney sum(WinningMoney firstOperand, WinningMoney secondOperand) {
```
이렇게도 바꿔 볼 수 있을 것 같아요 :)